common/metrics/client.go

```diff
@@ -61,15 +59,15 @@ func NewClient(scope tally.Scope, serviceIdx ServiceIdx) Client {
 		scopeTags := map[string]string{
 			OperationTagName: def.operation,
 		}
-		common.MergeDictoRight(def.tags, scopeTags)
+		mergeDictoRight(def.tags, scopeTags)
```

Comment:
This change is to break circle dependency because common.CheckEventBlobSizeLimit() in common package depends metrics package.